tester/src/test/java/com/silanis/esl/sdk/examples/AccountLimitSupportedLanguagesSettingsExampleTest.java

```diff
@@ -0,0 +1,47 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.api.model.SupportedLanguages;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class AccountLimitSupportedLanguagesSettingsExampleTest {
+
+    @Test
+    public void verifyResult() {
+        AccountLimitSupportedLanguagesSettingsExample example = new AccountLimitSupportedLanguagesSettingsExample();
+        example.run();
+        SupportedLanguages supportedLanguages1 = getSupportedLanguages(Arrays.asList("en", "fr"), "fr");
```

Comment:
Can you make the supportedLanguages1 & supportedLanguages2 private and place them outside the class ?